integration/network/network.go

```diff
@@ -0,0 +1,173 @@
+package network
```

Comment:
In general I'm confused. You provided nice description in the PR, but the code provides a lot of customizations which I doubt are necessary.

I like the idea of using job ID to create a subnet. That will allow us to run ~120 jobs in parallel (assuming the ID is increasing integer number). Why not just provide the CIDR depending on job ID and disallow all other options?

I'd like to know what is "host cluster" vnet that the peering is created with.

Reply:
What the unnecessary options/customizations in your opinion ?


https://github.com/giantswarm/azure-operator/blob/1df1de711fc8dbdb3593fc176949dcadbd385520/integration/template/azure_operator_charts_values.go#L19-L23

So the host cluster running in ci has: `CIDR: "10.0.0.0/16"`
But vnet peering is happening with godsmack